### DISCUSSION TITLE
Selecciona el Framework

### DISCUSSION THREAD

**Discussion by erykzon** [Launch-X-Latam/MisionFrontEnd#42] [Polls]
![react_original_logo_icon_146374](https://user-images.githubusercontent.com/69286881/192628018-a85193f9-1d60-45a6-8666-f9335d668b69.png)  ![vuejs_original_wordmark_logo_icon_146305](https://user-images.githubusercontent.com/69286881/192627679-81bf6342-e782-47d5-a8cc-9b7eb51c1d63.png)

**Comment by Ireneripo-LaunchX** [comment]
Hola, ya existe esta encuesta [acá](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/31#discussion-4424287).
